src/pages/workspace/WorkspaceCardPage.js

```diff
@@ -179,7 +179,6 @@ const WorkspaceCardPage = ({
                                     success
                                     large
                                     text={buttonText}
-                                    isLoading={reimbursementAccount.loading}
```

Comment:
Is there any case where this can be true, but the modal is not open?

Reply:
Nope, not that I know of / could figure out